internal/services/ingest_test.go

```diff
@@ -215,3 +216,166 @@ func TestIngestPayments(t *testing.T) {
 		assert.Equal(t, payments[0].TransactionHash, "abcd")
 	})
 }
+
+func TestIngestPathPaymentsSend(t *testing.T) {
```

Comment:
can you make this a unit test under the TestIngestPayments function? With a t.Run("op_pat_payment_send", func(t *testing.T) { ...} ? That way you can also reuse the same set up code